server/utils/utils.js

```diff
@@ -62,6 +62,17 @@ function getContentType(req) {
   return contentType;
 }
 
+function getNewIdFromDatabaseItems(items) {
+  if (items.length === 0) {
+    return 0;
+  }
+  if (items.length === 1) {
+    return items[0].id + 1;
+  }
+  const maxId = Math.max(...items.map(item => item.id));
```

Comment:
just to be safe let's make sure `item.id` is always a number, because in some cases it turned out to be a string